src/OrderLinesList/OrderLinesFiltersContainer.js

```diff
@@ -39,6 +46,7 @@ OrderLinesFiltersContainer.manifest = Object.freeze({
 OrderLinesFiltersContainer.propTypes = {
   activeFilters: PropTypes.object.isRequired,
   applyFilters: PropTypes.func.isRequired,
+  customFields: PropTypes.arrayOf(PropTypes.object),
```

Comment:
Could you add default props `customFields: []`?
I see that you added it in OrderLinesList.js file.

Reply:
Done. Opted for a default parameter as `defaultProps` are deprecated.